tests/test_load_from_msp.py

```diff
@@ -0,0 +1,42 @@
+import os
+import numpy
+from matchms.importing import load_from_msp
+
+
+def test_load_from_msp():
+    """ MSP file to a Spectrum object
+        Bellow is an example on how to parse a .msp file to a Spectrum object.
+
+    .. code-block:: python
+
+        import os
+        import numpy
+        from matchms.importing import load_from_msp
+
+        spectrum = load_from_msp("MoNA-export-GC-MS-first10.msp")
+
+        expected_inchikey = numpy.array(["ALRLPDGCPYIVHP-UHFFFAOYSA-N",
+            "UFBJCMHMOXMLKC-UHFFFAOYSA-N", "WDNBURPWRNALGP-UHFFFAOYSA-N", "RANCECPPZPIPNO-UHFFFAOYSA-N",
+            "HOLHYSJJBXSLMV-UHFFFAOYSA-N", "UMPSXRYVXUPCOS-UHFFFAOYSA-N", "HFZWRUODUSTPEG-UHFFFAOYSA-N",
+            "VPOMSPZBQMDLTM-UHFFFAOYSA-N", "LHJGJYXLEPZJPM-UHFFFAOYSA-N", "LINPIYWFGCPVIE-UHFFFAOYSA-N"])
+        for k, n in enumerate(spectrum):
+            assert n.get("inchikey") == expected_inchikey[k], "Expected different InChIKey."
```

Comment:
Nice code example. Just move it to the docstring of ``load_from_msp`` instead of here in the test.
Also, I would either add the part that requires ``os`` or remove the os import from the example.

Reply:
Thank you for letting me know. I changed the example to that file.